plugin/pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <analysis-model-api.version>9.3.2</analysis-model-api.version>
     <analysis-model-tests.version>${analysis-model-api.version}</analysis-model-tests.version>
-    <forensics-api-plugin.version>0.8.0-rc710.f9a5c103b329</forensics-api-plugin.version>
+    <forensics-api-plugin.version>0.8.0-rc719.d6b007b88023</forensics-api-plugin.version>
```

Comment:
@uhafner you should have a non-incremental release of all dependencies prior to merging anything to `master`.

Reply:
Ah, ok. Why is that? To always find a matching version of the dependency in the update center?

Reply:
Pending a safety belt in JENKINS-50804, to be absolutely sure you do not release this as is. The intent of JEP-305 is to offer well-defined artifacts for the duration of a set of matching PRs, with the dependencies converted to official versions as the upstream PRs are merged and released.

Reply:
Ok, thanks for clarification! I'm about to create a Christmas release so it will be fixed very soon...